feature/gnoi/system/tests/per_component_reboot_test/per_component_reboot_test.go

```diff
@@ -216,6 +218,12 @@ func TestLinecardReboot(t *testing.T) {
 		return true
 	})
 	if val, ok := watch.Await(t); !ok {
+		for port, status := range upInterfaces {
```

Comment:
Nit: I think "port" should be named as "intf", and "status" should be named as "up"?

Reply:
done